src/content/eol/2024/08/eol-08-01-24.md

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
we should probably delete your new WN post and redirect it to this page to avoid duplicative announcements

Reply:
I'll do that once this goes live.